privacyidea/static/components/audit/controllers/auditControllers.js

```diff
@@ -44,6 +44,8 @@ myApp.controller("auditController", function (AuditFactory, $scope,
         $scope.userFilter = $stateParams.user;
     }
 
+    $scope.$on("piReload", $scope.getAuditList);
```

Comment:
I think we need to move this line, as ``$scope.getAuditList`` is not yet defined at this place.

Reply:
see above.